### DISCUSSION TITLE
Contribution Graph

### DISCUSSION THREAD

**Discussion by igitasjay** [community/community#147927] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Can someone explain to me thoroughly why I make contributions and they do not reflect on my graph?
Sometimes the contributions that reflect are the ones that are made directly on GitHub, for instance, updating a Readme file counts and shows as a contribution...

**Comment by davevad93** [comment, marked as answer]
Hi @igitasjay ,

Normally, when contributions don't show up in the activity graph, it's for the following reasons:

1. The email isn't properly configured in `git config`.
2. Commits made in a secondary `branch` (not `main` or `master`). 
3. Commits made in a forked repo.
4. The `Include private contributions in my profile` option isn't checked in the `Settings`.

For more info see the docs:

 - [Why are my contributions not showing up on my profile?](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile)

**Reply by igitasjay** [reply]
Thanks a lot man.
Each one of your recommendations looked like an unlikely cause until I did git config --global user.email and saw how I erroneously configured it some 3 years ago when I didn't know how github works. 😂

**Reply by davevad93** [reply]
You're welcome @igitasjay 👍
